alcs-frontend/src/environments/environment.prod.ts

```diff
@@ -8,7 +8,7 @@ export const environment = {
   production: true,
   apiUrl: window.env?.['apiUrl'] || 'http://localhost:8080',
   homeUrl: window.env?.['homeUrl'] || '/home',
-  dateFormat: 'YYYY-MMM-dd',
+  dateFormat: 'YYYY-MMM-DD',
```

Comment:
'Th'/'Fr' fix?

Reply:
yep! Looks like I just forgot to update the prod file